src/smc-webapp/smc_chat.cjsx

```diff
@@ -485,7 +486,7 @@ ChatRoom = rclass ({name}) ->
             scroll_to_bottom(@refs.log_container, @props.actions)
 
     mark_as_read: ->
-        @props.redux.getActions('file_use').mark_file(@props.project_id, @props.path, 'read')
+        @props.redux.getActions('file_use').mark_file(@props.project_id, @props.path, 'seen', 0, false)
```

Comment:
What is going on here?  Why does a function called `mark_as_read` actually make the file as **seen**.  This can't possibly be right.  Either change the function name or what is done, or make a big comment explaining why the discrepancy.